public/locales/en/common.json

```diff
@@ -286,7 +286,7 @@
   "Podping Admin Tools": "Podping Administration",
   "Podverse logo": "Podverse logo",
   "Premium": "Premium",
-  "PremiumFreeTrial": "Premium - Free Trial",
+  "PremiumFreeTrial": "Premium — Gifted Period",
```

Comment:
Free Trial is commonly understood in the US, but Gifted Period is not. I'm going to change this back, and like I said in podverse-rn, I think maybe we need a context document for stuff like this.

Reply:
Think about whether you would want someone to get you a gifted period or a trial period. As they are effectively the same (and can be treated as such), the psychological effect is that Podverse itself gives something for which there is an incurred cost that the user feels inclined to repay in some fashion.
The user is then more likely to understand this to be a gift to others when getting it for others, instead of being indebted to the premise of the actual cost. "Free trial" is the language of trialware and shady practices, which doesn't exist anymore, and Podverse is not.